approval_polls/static/approval_polls/embed_instructions.js

```diff
@@ -0,0 +1,26 @@
+/* Code related to templates/approval_polls/embed_instructions.html */
+
+// Sets the iframe height and the corresponding embed instruction code in the textarea.
+function setIframeHeightAndCode(iframeId, textareaId, pollLink) {
+
+    // get the height of the contents of the iframe.
+    var iframeHeight = $(iframeId).contents().height();
```

Comment:
A very nice explanation. I am quite new to web development and hadn't come across this before. Thanks!